ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable.yaml

```diff
@@ -115,6 +115,20 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-f14-longduration-hive-mce-sd-rosa
```

Comment:
@huangmingxia job name should follow naming conversion spec https://docs.google.com/document/d/19PlQW82ryhOV652LsJrvDEFIDc4Z6HT0Aqp2G-8RmLY/edit?tab=t.0#heading=h.7tbkhyk5x3oz 
/cc @liangxia for more comments